message_definitions/v1.0/common.xml

```diff
@@ -4584,6 +4657,61 @@
       <field type="uint8_t" name="severity" enum="MAV_SEVERITY">Severity of status. Relies on the definitions within RFC-5424.</field>
       <field type="char[254]" name="text">Status text message, without null termination character.</field>
     </message>
+    <!-- Open Drone ID specific messages. The primary intention for these is to feed data to e.g. https://github.com/opendroneid/opendroneid-core-c -->
+    <message id="400" name="OPEN_DRONE_ID_BASIC_ID">
+      <wip/>
+      <!-- This message is work-in-progress it can therefore change, and should NOT be used in stable production environments -->
+      <description>Data for filling the OpenDroneID Basic ID message. See also https://github.com/opendroneid/opendroneid-core-c</description>
+      <field type="uint8_t" name="id_type" enum="OPENDRONEID_ID_TYPE">Indicates the format type for the ID of the UAS.</field>
```

Comment:
Maybe "for the uas_id" field of this message.

Reply:
Fixed.